### ISSUE TITLE
Make mars-sim accessible for visually disabled. 

### ISSUE THREAD

**Issue by shirishag75** [mars-sim/mars-sim#96]
Maybe we can make mars-sim accessible for visually disabled. Maybe in the long-term or in the short-term. 

Having speech readers would make it much easier for more people to enjoy the game. See https://wiki.debian.org/accessibility for some of the tools that could be used.

**Comment by mokun** [comment]
Yes it is very meaningful to pursue integrating each of these accessibility features. 

At some point, a voice control interface is very nice to have.

We are asking for more developers to join in for our efforts on many fronts.
